### PR TITLE
fix: Text in React Embed Code Becomes Invisible on Selection

### LINKED ISSUE #15832: Text in Embed Code Becomes Invisible on Selection
Found a bug? Please fill out the sections below. 👍

### Issue Summary

When selecting the provided embed code with the mouse, the selected lines become invisible (less visible) due to the text color and background color is same.

### Steps to Reproduce

1. Go to the Embed section of the calendar.
2. Try to select the text with the mouse.

### Actual Results

- On selection text is not visible.

### Expected Results

- When selected, the text should be visible.

### Evidence
Screenshot_2024-07-19_10_18_37

### PR DESCRIPTION
**Pull request by krishnendusukumar**
fixes #15832

### CODE REVIEW

**Review by anikdhabal — CHANGES_REQUESTED** [review]
Hey @krishnendusukumar pls uncommit the lock file  https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md#guidelines-for-committing-yarn-lockfile

File: packages/features/embed/lib/EmbedTabs.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const tabs = [
             data-testid="embed-code"
             ref={ref as typeof ref & MutableRefObject<HTMLTextAreaElement>}
             name="embed-code"
-            className="text-default bg-default h-[calc(100%-50px)] font-mono"
+            className="text-default h-[calc(100%-50px)] bg-gray-100 font-mono"
```
Comment (anikdhabal) [inline packages/features/embed/lib/EmbedTabs.tsx:47]: No changes needed here

File: packages/features/embed/lib/EmbedTabs.tsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ export const tabs = [
             data-testid="embed-react"
             ref={ref as typeof ref & MutableRefObject<HTMLTextAreaElement>}
             name="embed-react"
-            className="text-default bg-default selection:bg-subtle h-[calc(100%-50px)] font-mono"
+            className="text-default selection:bg-subtle h-[calc(100%-50px)] bg-gray-100 font-mono"
```
Comment (anikdhabal) [inline packages/features/embed/lib/EmbedTabs.tsx:99]: Only remove the `selection:bg-subtle`

Comment (krishnendusukumar) [reply packages/features/embed/lib/EmbedTabs.tsx:99]: oh okay my bad removing yarn.lock file.

**Review by anikdhabal — DISMISSED** [review]
Made changes
LGTM!

**Comment by anikdhabal** [comment]
@krishnendusukumar, did you not see the changes I made to this PR before approving it? I had already fixed everything, but you still committed the wrong changes.

**Review by anikdhabal — APPROVED** [review]

**Comment by krishnendusukumar** [comment]
i apologise for this, i was setting the project for the next issues and by mistake i did this

**Pull request merged**